exporters/otlp/trace/src/test/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterTest.java

```diff
@@ -83,13 +87,12 @@ void invalidConfig() {
     assertThatThrownBy(() -> OtlpGrpcSpanExporter.builder().setEndpoint(null))
         .isInstanceOf(NullPointerException.class)
         .hasMessage("endpoint");
-    assertThatThrownBy(() -> OtlpGrpcSpanExporter.builder().setEndpoint("😺://localhost"))
```

Comment:
sorry, no more kitty-cats. they'll have to fail when grpc tries to connect

Reply:
Aww thought we saved the cat 😿

Reply:
ah well.. we can bring it back if you think it's an important use-case that people might fall into.  ;)